orc8r/cloud/go/obsidian/swagger/v1/index.html

```diff
@@ -93,6 +103,19 @@
   window.ui = ui
 }
 </script>
+
+<script>
```

Comment:
What's this script doing?

Reply:
It's the control logic for highlighting the current service on the sidebar.

Reply:
@AndyLKhuu I feel like you should be able to accomplish this just with templating. Lmk if you think that won't work

Reply:
Discussed offline to move sidebar logic into templating